main/modes/mode_tunernome.c

```diff
@@ -353,6 +357,7 @@ static const char playStringsText[] = "Play all open strings";
 static const char listeningText[] = "Listening for a note";
 static const char leftStr[] = ": ";
 static const char rightStrTuner[] = "Start: Tuner";
+static const char rightStrTunerSel[] = "Sel: SubBeat";
```

Comment:
It would be nice if this listed the number of subbeats similar to how the number of beats are listed.